indico/modules/rb_new/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -193,6 +196,14 @@ class TimelineItem extends React.Component {
                     {!!rejectionReason && <div>{Translate.string('Reason: {reason}', {reason: rejectionReason})}</div>}
                 </div>
             );
+        } else if (type === 'conflicting-candidates') {
+            popupContent = (
+                <div styleName="popup-center">
+                    <strong>
+                        <Translate>Candidate is conflicting with other occurrences</Translate>
```

Comment:
in case of an exact overlap between the candidate and the other occurrence, do we still see details about the conflicting occurrence?

Reply:
If it renders those popups according to the renderOrder then it should be okay in all cases. I put conflictingCandidates in a higher order than candidates.

Reply:
but do we see e.g. the reason of the conflicting booking and who made it?